BAZProject/Modules/MainMovieModule/MainPresenter.swift

```diff
@@ -55,3 +96,92 @@ extension MainPresenter: MainInteractorOutputProtocol{
         view?.reloadData()
     }
 }
+
+extension MainPresenter: UITableViewDataSource, UITableViewDelegate {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return "Tendencia:"
+        case 1:
+            return "En cines:"
+        case 2:
+            return "Popular:"
+        case 3:
+            return "Mejor valoradas: "
+        case 4:
+            return "Proximamente: "
+        default:
+            return "not Found"
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: MoviesTableViewCell.reusableIdentifier) as? MoviesTableViewCell {
+            
+            switch indexPath.section {
+            case 0:
+                if let dataMovies = interactor?.movieApiData.getArrayDataMovie?[.trending] as? Movies {
```

Comment:
The only thing that changes in one case or another is the index for the dictionary. Thus, you could use the enum you already have instead of a switch case.

```suggestion
                if let dataMovies = interactor?.movieApiData.getArrayDataMovie?[Enum.indexForSection(for: indexPath.section)] as? Movies {
```

Reply:
In that way you can get rid of the switch